cfpb/sbl-frontend
# [Resolve Errors] [Review Warnings] Exceed error warning limit alert & counts read off the response body

closes #623
closes #624 
closes #630 
closes #704 

## Changes
- feat(Errors/Warnings): A field alert alert appears when the error/warnings limit is exceeded (i.e. `is_truncated`)
- feat(Errors/Warnings): Single Field, Multi Field and Register counts now come from the data response object
- enhancement(Download Validation Report): Added button loader
- enhancement(Field Level Alert - Download Validation Report Link): Now scrolls to the Download Validation Report button

## How to Test
- Upload [1_000_000 errors](https://github.com/cfpb/sbl-test-data/blob/main/locust-sblars/100000_entries_findings.csv)
- Navigate to Errors and/or Warnings 
- Ensure the counts properly match the data response object
- The Field Level Alert appears when necessary
- Test the Download Validation report link 

## Screenshot
<img width="623" alt="Screenshot 2024-06-12 at 2 50 47 PM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/6860f393-dcec-4332-8aa6-f94481b8ddfd">
<img width="623" alt="Screenshot 2024-06-12 at 2 50 47 PM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/6860f393-dcec-4332-8aa6-f94481b8ddfd">
<img width="623" alt="Screenshot 2024-06-12 at 2 50 47 PM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/6860f393-dcec-4332-8aa6-f94481b8ddfd">
